metadata.rb

```diff
@@ -18,4 +18,4 @@
 supports 'ubuntu'
 
 chef_version '>= 15.0'
-gem 'docker-api', '~> 1.34.0'
```

Comment:
This should be >= 1.34, < 3 probably so it works with older versions of chef correctly